### PR TITLE
sgt-back-end | Rich Trunzo 

### PR DESCRIPTION
**Pull request by richtrunzo**
Screen Shot 2021-01-14 at 9 33 04 AM
Screen Shot 2021-01-14 at 9 35 14 AM
Screen Shot 2021-01-14 at 9 36 28 AM
Screen Shot 2021-01-14 at 9 36 50 AM
Screen Shot 2021-01-14 at 9 37 22 AM
Screen Shot 2021-01-14 at 9 39 53 AM
Screen Shot 2021-01-14 at 9 40 15 AM
Screen Shot 2021-01-14 at 9 40 31 AM
Screen Shot 2021-01-14 at 9 41 22 AM
Screen Shot 2021-01-14 at 9 42 16 AM
Screen Shot 2021-01-14 at 9 42 37 AM
Screen Shot 2021-01-14 at 9 42 47 AM
Screen Shot 2021-01-14 at 9 43 17 AM

### CODE REVIEW

File: sgt-back-end/index.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+app.get('/api/grades', (req, res, next) => {
+  const sql = `select *
+              from "grades"`;
+db.query(sql)
+  .then(result => {
+    res.status(200).json(result.rows);
+  })
+  .catch(err => {
+    console.log(err);
+    res.status(500).json({'error': 'An unexpected error occured'});
```
Comment (BrettAlbright) [inline sgt-back-end/index.js:23]: make sure they know it was a server error.

File: sgt-back-end/index.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+  const sql = `insert into "grades" ("name", "course", "score")
+               values ($1, $2, $3)
+               returning *`;
+  const params = [name, course, score];
+  db.query(sql, params)
+  .then(result => {
+    res.status(200).json(result.rows[0]);
+  })
+  .catch(err => {
+    res.status(500).json({error: `An unexpected error occured`});
```
Comment (BrettAlbright) [inline sgt-back-end/index.js:44]: same as above

**Review by BrettAlbright — APPROVED** [review]
👍

File: sgt-back-end/index.js (hunk 2 above)
Comment (richtrunzo) [reply sgt-back-end/index.js:44]: Got it thank you!

**Pull request merged**